packages/aws-sdk-client-mock-jest/vitest.serializer.ts

```diff
@@ -0,0 +1,21 @@
+import { appendFileSync } from "fs";
+import prettyAnsi from "pretty-ansi";
+import type { SnapshotSerializer } from "vitest";
+
+export default {
+  serialize(text, config, indentation, depth, refs, printer) {
+    appendFileSync("snapshot.log", text + "\n")
```

Comment:
Do we need it since all snapshots we compare to are inline?

Reply:
Deleted it.